test/integration/narrow/routes/utils/show-multi-herds-banner.test.js

```diff
@@ -0,0 +1,75 @@
+import { showMultiHerdsBanner } from "../../../../../app/routes/utils/show-multi-herds-banner";
+import { config } from "../../../../../app/config/index.js";
+
+const { multiHerds } = config;
+
+jest.mock("../../../../../app/config/index");
+multiHerds.enabled = true;
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+test("no applications", () => {
```

Comment:
It's no real biggie, but a slightly more descriptive tone that lays out the intention of the test might help in case of future debug if they ever started to fail after a change later on. For instance 'should not show banner if no previous applications' might be slightly better here (and same for other examples below)

Reply:
yeah fair point - they were just copy and pasted from the previous test file (test/integration/narrow/routes/utils/user-needs-notification.test.js). I'll make the names better though, you're right!